src/tests/edge_ids/saving_transitionid.rs

```diff
@@ -0,0 +1,60 @@
+#[cfg(test)]
+mod saving_transitionid_test {
+    use crate::System::save_component::{combine_components, PruningStrategy};
+    use crate::{
+        tests::reachability::helper_functions::reachability_test_helper_functions,
+        ModelObjects::representations::QueryExpression,
+    };
+    use std::collections::HashSet;
+    use std::iter::FromIterator;
+    use test_case::test_case;
+    const FOLDER_PATH: &str = "samples/json/EcdarUniversity";
+
+    #[test_case(FOLDER_PATH, QueryExpression::VarName("Machine".to_string()), vec![
+        "E0".to_string(),
+        "E1".to_string(),
+        "E2".to_string(),
+        "E3".to_string(),
+        "E4".to_string()]; "Simple save component transition id test")]
+    #[test_case(FOLDER_PATH,
+        QueryExpression::Conjunction(
+            Box::new(QueryExpression::VarName("HalfAdm1".to_string())),
+            Box::new(QueryExpression::VarName("HalfAdm2".to_string()))),
+        vec![
+            "E0".to_string(),
+            "E1".to_string(),
+            "E2".to_string(),
+            "E3".to_string(),
+            "E4".to_string(),
+            "E5".to_string(),
+            "E6".to_string(),
+            "E7".to_string(),
+            "E8".to_string(),
+            "E9".to_string(),
+            "E10".to_string(),
+            "E11".to_string()
+            ]; "Conjunction save HalfAdm1 and HalfAdm2")]
+    fn transition_save_id_checker(
+        path: &str,
+        machineExpression: QueryExpression,
+        transition_ids: Vec<String>,
+    ) {
+        let mock_model = Box::new(machineExpression);
+        let mut expected_ids: HashSet<&String> = HashSet::from_iter(transition_ids.iter());
+        let (_, system) =
+            reachability_test_helper_functions::create_system_recipe_and_machine(*mock_model, path);
```

Comment:
Synes der skal stå FOLDER_PATH istedet for path

Reply:
done